lib/member_page.rb

```diff
@@ -25,14 +25,26 @@ class MemberPage < Scraped::HTML
     party_data.last
   end
 
+  field :party_wikidata_id do
+    party_identifier = party_node.at_xpath('.//a/@data-identifier-wikidata')
+    return nil if party_identifier.to_s.empty?
+    party_identifier.text
+  end
+
   field :area do
     area = sidebar.xpath('.//h3[.="Province"]/following-sibling::ul[1]/li').text.tidy
     return 'National' if area.to_s.empty?
     area
   end
 
+  field :area_wikidata_id do
+    id = sidebar.xpath('.//h3[.="Province"]/following-sibling::ul[1]/li/a/@data-identifier-wikidata').text.tidy
```

Comment:
Would be good to clear up the duplication between this and the `area` field. Perhaps you could extract an `area_node` method similar to `party_node`?